ignite/distributed/utils.py

```diff
@@ -489,14 +493,20 @@ def train_fn(local_rank, a, b, c):
         backend: backend: `nccl`, `gloo`, `xla-tpu`, `horovod`.
         kwargs: acceptable kwargs according to provided backend:
 
-            - "nccl" or "gloo" : timeout(=timedelta(minutes=30)).
+            - | "nccl" or "gloo" : ``timeout(=timedelta(minutes=30))``, ``init_method(=None)``,
+              | ``rank(=None)``, ``world_size(=None)``.
+              | By default, ``init_method`` will be "env://". See more info about parameters: `torch_init`_.
 
-            - "horovod" : comm(=None), more info: `hvd_init`_.
+            - | "horovod" : comm(=None), more info: `hvd_init`_.
 
+    .. _torch_init: https://horovod.readthedocs.io/en/latest/api.html#horovod.torch.init
```

Comment:
The two links are the same , is this the intention?

Reply:
Does documentation work if one of them removed ?

Reply:
No it doesn't , I'm just not sure that the first link should be pointing to horovod docs , shouldn't it point to pytorch distributed?

Reply:
Good remark, yes, it is a wrong copy-paste. It should point out to pytorch docs. Thanks !